pkg/agent/config.go

```diff
@@ -177,7 +177,7 @@ func ParseConfig(data []byte, isVenafiCloudMode bool) (Config, error) {
 
 	if config.Server == "" && config.Endpoint.Host == "" && config.Endpoint.Path == "" {
 		config.Server = "https://preflight.jetstack.io"
```

Comment:
this should be the devstack url? defined in the agent yaml config?

Reply:
This URL is for the standalone backend. If you want to define a different URL for the devstack, you just need to add this at the top of your configuration.

```yaml
server: <devstack url>
```